### PR TITLE
Frontier - Add GALAXY token

### PR DESCRIPTION
**Pull request by jayjay-crypto**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -3006,6 +3012,48 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
+			coinDenom: 'GLX',
+			coinMinimalDenom: 'uglx',
+			coinDecimals: 6,
+			coinGeckoId: '',
+			coinImageUrl: window.location.origin + '/public/assets/tokens/glx.svg',
+		},
+		bip44: {
+			coinType: 118,
+		},
+		bech32Config: Bech32Address.defaultBech32Config('galaxy'),
```
Comment (JeremyParish69) [inline src/config.ts:3030]: Both here, and on the Chain Registry, bech32 prefix is 'galaxy', while registered to SLIP0173, it is 'glx'. Which is correct? if 'galaxy', please have the SLIP-173 registry corrected.
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: src/config.ts [hunk 2]
```diff
@@ -1162,6 +1162,12 @@ export const IBCAssetInfos: {
 		destChannelId: 'channel-10',
 		coinMinimalDenom: 'afet',
 	},
+    {
```
Comment (JeremyParish69) [inline src/config.ts:1165]: This should be at the end of the list.

File: src/config.ts [hunk 3]
```diff
@@ -3006,6 +3012,48 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
 		explorerUrlToTx: 'https://www.mintscan.io/fetchai/txs/{txHash}',
 	},
+    {
```
Comment (JeremyParish69) [inline src/config.ts:3015]: Although the ordering doesn't really matter here, it'd be easier to find if this were at the end of the list. Can we move it down?

File: src/config.ts (hunk 1 above)
Comment (jayjay-crypto) [reply src/config.ts:3030]: I have mixed in slip. correction made

File: src/config.ts (hunk 3 above)
Comment (jayjay-crypto) [reply src/config.ts]: Yes we can. I've move to the end.

File: src/config.ts [hunk 4]
```diff
@@ -3006,6 +3012,48 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
+		rpc: 'https://rpc.galaxychain.zone',
+		rest: 'https://rest.galaxychain.zone',
+		chainId: 'galaxy-1',
+		chainName: 'galaxy',
+		stakeCurrency: {
+			coinDenom: 'GLX',
+			coinMinimalDenom: 'uglx',
+			coinDecimals: 6,
+			coinGeckoId: '',
```
[the remark this answers is not in the document]
Comment (jayjay-crypto) [reply src/config.ts]: done

File: src/config.ts [hunk 5]
```diff
@@ -3017,4 +3023,46 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
 		explorerUrlToTx: 'https://www.mintscan.io/provenance/txs/{txHash}',
 	},
+	{
+		rpc: 'https://rpc.galaxychain.zone',
+		rest: 'https://rest.galaxychain.zone',
+		chainId: 'galaxy-1',
+		chainName: 'galaxy',
```
Comment (JeremyParish69) [inline src/config.ts:3030]: This 'g' should be capitalized... unless it's meant not to be

File: src/config.ts [hunk 6]
```diff
@@ -3017,4 +3023,46 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
 		explorerUrlToTx: 'https://www.mintscan.io/provenance/txs/{txHash}',
 	},
+	{
+		rpc: 'https://rpc.galaxychain.zone',
```
Comment (JeremyParish69) [inline src/config.ts:3027]: I think wss needs to be enabled
image

Comment (jayjay-crypto) [reply src/config.ts:3027]: I going to ping the team. Back to you when it's ok

File: src/stores/root.ts [hunk 7]
```diff
@@ -1618,6 +1625,21 @@ export class RootStore {
 					},
 				],
 			},
+            {
+				poolId: '697',
```
Comment (JeremyParish69) [inline src/stores/root.ts:1629]: please ensure the spot price of this pool is appropriate

File: src/config.ts (hunk 6 above)
Comment (jinhopkg) [reply src/config.ts:3027]: @JeremyParish69 
'wss://rpc.galaxychain.zone/websocket' It works now.

File: src/stores/root.ts (hunk 7 above)
Comment (jinhopkg) [reply src/stores/root.ts:1629]: @JeremyParish69 
current spot price is correct.:)

File: src/config.ts (hunk 6 above)
Comment (JeremyParish69) [reply src/config.ts:3027]: no wss error anymore, but I do get this. Looks like it's being blocked for this staging site.
image

Comment (jinhopkg) [reply src/config.ts:3027]: when i deposit glx at frontier.osmosis.zone
The following error occurred 'network error'

tx details:
{
  "chain_id": "galaxy-1",
  "account_number": "4009",
  "sequence": "42",
  "fee": {
    "gas": "130000",
    "amount": [
      {
        "denom": "uglx",
        "amount": "3250"
      }
    ]
  },
  "msgs": [
    {
      "type": "cosmos-sdk/MsgTransfer",
      "value": {
        "source_port": "transfer",
        "source_channel": "channel-0",
        "token": {
          "denom": "uglx",
          "amount": "1000000"
        },
        "sender": "galaxy1yh4py3f0mym63z7v57krfljnac8ls39akaq02m",
        "receiver": "osmo1yh4py3f0mym63z7v57krfljnac8ls39aze4lhm",
        "timeout_height": {
          "revision_number": "1",
          "revision_height": "4284159"
        }
      }
    }
  ],
  "memo": ""
}

Comment (jinhopkg) [reply src/config.ts:3027]: @JeremyParish69
스크린샷 2022-05-06 오후 8 25 25

Comment (jayjay-crypto) [reply src/config.ts:3027]: Hi @jinhopkg I've just try deposit on frontier all goes smooth
image

**Pull request merged**